kitsune/wiki/strategies.py

```diff
@@ -296,6 +300,49 @@ def translate(self, l10n_request: TranslationRequest) -> TranslationResult:
         result.method = TranslationMethod.HYBRID
         return result
 
+    def reject_stale_translations(self) -> list[Revision]:
+        """
+        Reject stale, unreviewed machine translations.
+        """
+        if not settings.HYBRID_ENABLED_LOCALES:
+            return []
+
+        results = []
+        for rev in self.content_manager.get_stale_translations(
+            locales=settings.HYBRID_ENABLED_LOCALES
+        ):
+            self.content_manager.reject_revision(rev, comment="No longer relevant.")
+            results.append(rev)
+        return results
+
+    def publish_unreviewed_within_grace_period(self) -> list[Revision]:
+        """
+        Publish fresh machine translations that have not been reviewed within the grace period.
+        """
+        if not (settings.HYBRID_REVIEW_GRACE_PERIOD and settings.HYBRID_ENABLED_LOCALES):
+            return []
+
+        results = []
+        for rev in self.content_manager.get_fresh_translations(
```

Comment:
The query should be part of this method - similar comment was made for the manager to handle single revisions